tests/Aspire.Azure.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/WorkaroundToReadProtectedField.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.EntityFrameworkCore;
+using Npgsql.EntityFrameworkCore.PostgreSQL;
+
+namespace Aspire.Azure.Npgsql.EntityFrameworkCore.PostgreSQL.Tests;
+
+public class WorkaroundToReadProtectedField : NpgsqlRetryingExecutionStrategy
+{
+    public WorkaroundToReadProtectedField(DbContext context) : base(context)
```

Comment:
Can we get around a bunch of this duplication if we referenced the existing Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests project?